app/src/main/res/drawable/ic_baseline_square_unchecked2.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="72"
```

Comment:
simplify it - no need for square_unchecked with a numbers - for just color changing